src/open/clients/Cinny.js

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Maturity, Platform, LinkKind, FlathubLink, AppleStoreLink, PlayStoreLink, WebsiteLink } from "../types.js";
+
+/**
+ * Information on how to deep link to a given matrix client.
+ */
+export class Cinny {
+    get id() { return "in.cinny"; }
+    get name() { return "Cinny"; }
+    get icon() { return "images/client-icons/cinny.svg"; }
+    get author() { return "ajbura"; }
+    get homepage() { return "https://cinny.in"; }
+    get platforms() {
+        return [
+            Platform.DesktopWeb,
+        ];
+    }
+    get description() { return "Yet another Matrix client."; }
+    getMaturity(platform) {
+        switch (platform) {
+            case Platform.Android: return Maturity.Stable;
```

Comment:
You probably shouldn't put Android and iOS as Stable and then later say it doesn't support Android. Just leave those unsupported platforms out.

Reply:
oh whoops